lerna.json

```diff
@@ -9,5 +9,11 @@
       "message": "chore: publish [skip ci]",
       "conventionalCommits": true
     }
-  }
+  },
+  "ignoreChanges": [
+    "**/*.md",
+    "**/*.spec.*",
+    "**/test/**",
+    "packages/*/package-lock.json"
```

Comment:
What's a situation where package-lock.json changes and we don't want to publilsh a new version?

Reply:
The `package-lock.json` does not get published when publishing a npm package. Only the `package.json` is published.

https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json
> One key detail about package-lock.json is that it cannot be published